server.js

```diff
@@ -15,13 +15,22 @@ const express = require('express');
 const app = express();
 
 const port = process.env.PORT || 3000;
+
+// HTTP request for pinging
+//
+// Potentially used for keeping bot alive.
+app.get('/ping', function(req, res){
+   res.send("pong");
+});
+
```

Comment:
delete unused route

Reply:
I thought we could keep it for later, but sure.

Reply:
Deleted, see f8c383f.